packages/api/src/tx/msg.ts

```diff
@@ -37,7 +37,7 @@ export async function setupTxExtension(
   messageTypeRegistry.forEach((value, key) => {
     customRegistry.push([`/${key}`, value]);
   });
-  regenRegistry.forEach(([key, value]) => {
+  regenTypeRegistry.forEach(([key, value]) => {
     customRegistry.push([`/${key}`, value]);
   });
```

Comment:
not related to this PR, but why do we need this?
`messageTypeRegistry.forEach` above is already registering all the types

Reply:
deleted